scrunch/tests/test_utilities.py

```diff
@@ -133,14 +133,10 @@ def test_get_dataset_from_project(self, session):
         projects.entity.datasets.by.side_effect = _by_side_effect(
             shoji_entity, ds_res)
 
-        ds = get_dataset('dataset_name', project='project_name')
```

Comment:
We should also return this test to its previous state, and add a test for a get_dataset that passes a dataset id that is inside a project.